atlp-rwanda/script-heroes-frontend
# #174797468 Implement requests table

**What does this PR do?**
This PR displays users requests with details about them

**Description of Task to be completed?**
- create a component for displaying all requests
- create a component for displaying a singl request details
- test the feature 

**How should this be manually tested?**

- clone this repo `git clone https://github.com/atlp-rwanda/script-heroes-frontend`
- checkout the branch git checkout `ft-implement-requests-table-#174797468`
- install dependencies` yarn install`
- start the app yarn run dev
- login` /login`
- got to` /requests` to display all requests 
- got to` /requests/:id` to display a particular request details
- check responsiveness on different screen sizes


**What are the relevant pivotal tracker stories?**
   [#174797468](https://github.com/atlp-rwanda/script-heroes-frontend/pulls)

**Screenshots (if appropriate)**
![Screenshot from 2020-12-14 09-39-30](https://user-images.githubusercontent.com/58434968/102053630-b6029500-3df0-11eb-94b3-fa4172c89b6d.png)

![Screenshot from 2020-12-07 10-24-31](https://user-images.githubusercontent.com/58434968/101326954-ae823f80-3876-11eb-9f96-489c76effd20.png)
